src/libopensc/card-openpgp.c

```diff
@@ -327,10 +338,11 @@ pgp_match_card(sc_card_t *card)
 {
 	int i;
 
+	LOG_FUNC_CALLED(card->ctx);
```

Comment:
The convention in the existing code base is to *not* log entering/exiting `match_card()`, please stick with it.